examples/nft_improved_worker_gltf/threejs_worker.js

```diff
@@ -6,6 +6,25 @@ function isMobile() {
     return /Android|mobile|iPad|iPhone/i.test(navigator.userAgent);
 }
 
+const frameLength = 200; // in ms
```

Comment:
@hatsumatsu maybe this is not needed? i can not see where is used in your code https://github.com/hatsumatsu/lab--ar001/blob/5232a099cd0729dc19d79e2d97573f17873045df/js/app.js#L5

Reply:
i mean `frameLenght`

Reply:
not needed anymore.